doc/examples/applications/plot_3d_image_processing.py

```diff
@@ -0,0 +1,294 @@
+"""
+============================
+Explore 3D images (of cells)
+============================
+
+This tutorial is an introduction to three-dimensional image processing. Images
+are represented as `numpy` arrays. A single-channel, or grayscale, image is a
+2D matrix of pixel intensities of shape `(row, column)`. We can construct a 3D
```

Comment:
since you are porting markdown to rst, there are several places like here where you need the text to be enclosed in double backticks instead of single ones.

Reply:
Oh, I didn't notice because it rendered nicely (sort of):
![backticks](https://user-images.githubusercontent.com/2227806/87968307-80b37e80-cac0-11ea-8f72-aeb1344fbf1d.png)
now I realize this is italic, not code-style!

Reply:
indeed :-).

Reply:
but are you ok changing the single backticks to double ones?

Reply:
I'm definitely ok, in the sense that I don't have the strongest opinion. But I have hesitations in terms of consistency and pedagogy: Words in code style should refer to words... in code. For example, "the shape `(plane, row, column)`" (lines 9--10) should match the code we have line 98:
```py
- (n_plane, n_row, n_col) = data.shape
+ (plane, row, column) = data.shape
```
There is some confusion because we are using "(plane, row, column)" to mean:
- the number of planes, rows, columns (shape);
- the dimensions (axes) as in line 34;
- the indices (coordinates) along the plane, row, column dimensions as in the summary table, line 16.
In this context, I think that using italic style for one of the meanings not found in code would make sense.

Reply:
Ok, I went for the following: e8c23f90. I hope it's not too heavy in terms of wording...
In the table, coordinates use the square-bracket notation, as they should in general and recalling/announcing code line 103--105.
Alternatively, I considered keeping the italic style to describe dimensions:
![dimensions](https://user-images.githubusercontent.com/2227806/88584263-ac4edf80-d051-11ea-9528-15c4f8adf5f9.png)
which could also make sense.